tmail-backend/tmail-third-party/openpaas/src/main/java/com/linagora/tmail/dav/DavClient.java

```diff
@@ -174,11 +173,10 @@ private Mono<Void> doUpdateCalendarObject(String username, DavCalendarObject upd
             .responseSingle((response, responseContent) -> handleCalendarObjectUpdateResponse(updatedCalendarObject, response));
     }
 
-    @VisibleForTesting
     public Mono<Void> createCalendar(String username, URI uri, Calendar calendarData) {
         return client.headers(headers -> headers.add(HttpHeaderNames.CONTENT_TYPE, "text/plain")
                 .add(HttpHeaderNames.AUTHORIZATION, authenticationToken(username)))
-            .request(HttpMethod.PUT)
+            .request(HttpMethod.valueOf("MKCALENDAR"))
```

Comment:
Can you add basic tests for `createCalendar` ` deleteCalendar` method 
in the `CalDavEventRepositoryTest` ?
It is using "real" openpaas extension (not mock)